core/modules/parsers/wikiparser/rules/codeblock.js

```diff
@@ -11,7 +12,28 @@ Wiki text rule for code blocks. For example:
 	```
 ```
 
+@module $:/core/modules/parsers/wikiparser/rules/codeblock.js
```

Comment:
Perhaps we should extend the JS deserialiser to recognise `@module ` as a synonym for `title: `?

Reply:
This is not working, while this usage exists. I need to figure out why.